netprofile/development.ini

```diff
@@ -183,6 +183,10 @@ netprofile.ext.direct.debug_mode = true
 #       server.
 netprofile.rt.host = localhost
 
+# Url to connect to realtime server, if you use one.
+#
```

Comment:
Empty line not needed. Also, this needs to be in production.ini template.
Text is a bit misleading. Might be better to say something like "URL of the realtime server, if it differs from host/port provided above." (so move this below netprofile.rt.port).